### PR TITLE
deps: update trip-details

### PR DESCRIPTION
**Pull request by daniel-heppner-ibigroup**
Updates trip details to support the new OTP2 fare table.

### CODE REVIEW

File: lib/components/narrative/line-itin/connected-itinerary-body.js [hunk 1]
```diff
@@ -37,6 +37,11 @@ const noop = () => {}
 
 const ItineraryBodyContainer = styled.div`
   padding: 0px 10px;
+
+  /* TODO: Find a better way of doing this */
+  table {
+    font-size: 14px;
```
Comment (miles-grant-ibigroup) [inline lib/components/narrative/line-itin/connected-itinerary-body.js:43]: I don't think this is as much of a hack as you think it is!

**Review by miles-grant-ibigroup — APPROVED** [review]
Works and looks great!

File: lib/components/narrative/line-itin/connected-itinerary-body.js (hunk 1 above)
Comment (daniel-heppner-ibigroup) [reply lib/components/narrative/line-itin/connected-itinerary-body.js:43]: Should probably add a class name to the table someday

Comment (miles-grant-ibigroup) [reply lib/components/narrative/line-itin/connected-itinerary-body.js:43]: I think since this is styled components you don't need to!

**Review by demory — APPROVED** [review]

**Pull request merged**